docs/source/daml/intro/1_Token.rst

```diff
@@ -34,17 +34,17 @@ DAML specifies what transactions are legal on a DAML Ledger. The rules the DAML
 DAML files and modules
 ----------------------
 
-Each ``.daml`` file defines a *DAML Module*. At the top of each DAML file is a pragma informing the compiler of the language version and the module name:
+Each ``.daml`` file defines a *DAML Module* at the top:
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 6, 9
+  :lines: 5, 9
```

Comment:
I manually verified because no idea what `9` means here

Reply:
It's a list of lines. This means "show lines 5 and 9".

Reply:
oh. then `6` was a mistake, so now `9` is a mistake, both were/are empty lines ignored by doc builder :)
so changing to 
```suggestion
  :lines: 5
```